platforms/corda-ent/charts/notary-initial-registration/values.yaml

```diff
@@ -77,40 +55,102 @@ vault:
   # Eg. serviceaccountname: vault-auth
   serviceaccountname:
   # Provide the vault path where the  certificates are stored
-  # Eg. certsecretprefix: secret/cenm-org-name
+  # Eg. certsecretprefix: secret/organisation-name
   certsecretprefix:
- 
-
-# Provide volume related specifications
-volume:
- # Ex baseDir: /opt/corda
-  baseDir:
-
-# debug option for the bash scripts
-# Ex bashDebug: false
-bashDebug:
-
-cordaJarMx: 
-devMode: false
-p2pPort: 10002
-
-rpcSettingsAddress: "0.0.0.0"
-rpcSettingsAddressPort: 10003
-rpcSettingsAdminAddress: "localhost"
-rpcSettingsAdminAddressPort: 10770
-rpcSettingsStandAloneBroker: false
-rpcSettingsUseSsl: false
+  # Provide the vault orginsation node folder name where the certificates are stored
+  # Eg. nodePath: notary
+  nodePath:
+  # Number of retries to check contents from vault 
+  retries:
+  # Interval in seconds between retries
+  retryInterval:
+
+#############################################################
+#                    Notary Configuration                   #
+#############################################################
+nodeConf:
+  p2p:
+    url:
+  ambassador:
+    p2pPort: 
+    external_url_suffix: 
+    p2pAddress: 
+  # Notary legal name
+  # Eg. legalName: O=Notary,OU=Notary,L=London,C=GB
+  legalName:
+    # Email address
+    # Eg. emailAddress: dev-node@baf.com
+    emailAddress: 
+    # Notary is validator Ex : isValidating: true
+  notary:
+    validating:
+    # Dev Mode
+  devMode: false
+  # Provide volume related specifications
+  volume:
+    # Ex baseDir: /opt/corda
+    baseDir:
+  # where is node jar is stored
+  jarPath: 
+  # where service configuration files are stored
+  configPath:
+  # Specify the maximum size of the memory allocation pool
+  cordaJar:
+    # Provide the memory size.
+    # Eg. memorySize: 4096 (if using kilobytes)
+    # Eg. memorySize: 512 (if using megabytes)
+    # Eg. memorySize: 1 (if using gigabytes) 
+    memorySize:
+    # Provide the unit of greatness for the size, one of three options:
+    # - k or K for kilobytes
+    # - m or M for megabytes
+    # - g or G for gigabytes
+    # Eg. unit: M 
+    unit:
+    # Set limits of .jar  
+  pod:
+    resources:
+      # Provide the limit memory for node
+      # Eg. limits: 512M
+      limits:
+      # Provide the requests memory for node
+      # Eg. requests: 550M
+      requests:
+# Name of the cenm services
+# Ex. idmanName: idman
+# Ex. networkmapName: networkmap
+cenmServices:  

```

Comment:
cenm and network services should be together

Reply:
sorry had closed this.. will create new PR and to get these fixed